nav2_dwb_controller/dwb_core/include/dwb_core/illegal_trajectory_tracker.hpp

```diff
@@ -66,11 +66,11 @@ class IllegalTrajectoryTracker
  * @brief Thrown when all the trajectories explored are illegal
  */
 class NoLegalTrajectoriesException
```

Comment:
> Should these be entirely removed and replaced with nav2_msgs versions?

I don't think so. If you look below it has a specific illegal trajectory tracker for DWB. Once you have MPPI in we could re-evaluate and see if this is something we would like to pull out. Right now it is tied to the critics.